### PR TITLE
integration test results

### PR DESCRIPTION
**Pull request by jaycarlton**
Set up the test results pane in CircleCI using `store_test_results` step.

[Here's](https://app.circleci.com/pipelines/github/all-of-us/workbench/186/workflows/7f4fb993-6872-4957-8d57-2a1cf2f31e17/jobs/78110/tests) what a failing test looks like.

Test tab:
Screen Shot 2020-04-19 at 6 40 53 PM

With no errors:
Screen Shot 2020-04-19 at 7 24 42 PM

Steps tab:
Screen Shot 2020-04-19 at 6 41 05 PM

I haven't found where all our test results are written. Just integration tests so far.

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -297,6 +300,8 @@ jobs:
           name: Run e2e tests (Puppeteer and Jest)
           working_directory: ~/workbench/e2e
           command: yarn test:ci $(circleci tests glob "tests/**/*.spec.ts" | circleci tests split --split-by=timings --timings-type=classname --show-counts)
+      - store_test_results:
+          path: ~/workbench/api/build/test-results/integrationTest/
```
Comment (aweng98) [inline .circleci/config.yml:304]: Are you sure this works for the e2e test results? The e2e (JUnit) test results are explicitly set to write to `/e2e/logs` dir. And, `store_test_results` key is called again at line #308. I don't think circleci allows duplicate `store_test_results` keys in same job.

**Comment by jaycarlton** [comment]
I haven't tried to invoke the e2e on circle yet, so I'm not sure. I see you
have store_artifacts, which should populate the ARTIFACTS tab and I believe
using  store_test_results will make the TESTS tab work.

https://circleci.com/docs/2.0/configuration-reference/#store_test_results
https://circleci.com/docs/2.0/artifacts/

**Comment by aweng98** [comment]
LGTM! I didn't try run it.

File: .circleci/config.yml (hunk 1 above)
Comment (jaycarlton) [reply .circleci/config.yml]: whoops. good catch.

Is the TESTS UI tab working for this workflow?

**Comment by jaycarlton** [comment]
If we short circuit, the tests don't run, no results upload is attempted, and we still see the default message about `store_test_results` on the TESTS tab. This confused me a bit.

**Pull request merged**